tests/test_simple_put.cpp

```diff
@@ -172,7 +172,7 @@ void test_simple_put_one_byte(void **unused)
 
 static int gr_setup(void **unused)
 {
-  initialize_test(SF_BOOLEAN_FALSE);
+  initialize_test(SF_BOOLEAN_TRUE);
```

Comment:
what is this parameter?

Reply:
Will change to back to false. Basically this is used to control whether log will be printed to console or not. I enabled it for debugging purpose, but forget to change it back.